src/operator/nn/mkldnn/mkldnn_base-inl.h

```diff
@@ -178,7 +178,8 @@ struct SoftmaxOutputParam;
 struct TransposeParam;
 bool SupportMKLDNNAct(const ActivationParam& param);
 bool SupportMKLDNNAct(const ActivationParam& param, const NDArray &input);
-bool SupportMKLDNNConv(const ConvolutionParam& params, const NDArray &input);
+bool SupportQuantizedMKLDNNAct(const ActivationParam &param);
```

Comment:
could we resue `SupportMKLDNNAct`?

Reply:
we can't at moment. `SupportQuantizedMKLDNNAct` is a subset of `SupportMKLDNNAct`